underfs/oss/src/test/java/alluxio/underfs/oss/OSSOutputStreamTest.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.oss;
+
+import com.aliyun.oss.OSSClient;
+import com.aliyun.oss.OSSException;
+import com.aliyun.oss.model.ObjectMetadata;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.security.DigestOutputStream;
+
+/**
+ * Unit tests for the {@link OSSOutputStream}.
+ */
+@RunWith(PowerMockRunner.class)
+public class OSSOutputStreamTest {
+  private OSSClient mOssClient;
+  private File mFile;
+  private BufferedOutputStream mLocalOutputStream;
+
+  /**
+   * The exception expected to be thrown.
+   */
+  @Rule
+  public final ExpectedException mThrown = ExpectedException.none();
+
+  /**
+   * Sets the properties and configuration before each test runs.
+   *
+   * @throws Exception when the {@link HttpURLConnection#getOutputStream()} throws exception
+   */
+  @Before
+  public void before() throws Exception {
+    mOssClient = PowerMockito.mock(OSSClient.class);
```

Comment:
@albericliu Generally it's better to use Mockito instead of PowerMockito when possible, and we can do
```java
mOssClient = Mockito.mock(OSSClient.class);
mFile = Mockito.mock(File.class);
mLocalOutputStream = Mockito.mock(BufferedOutputStream.class);
```